common-test-lib/src/main/java/com/google/gct/idea/testing/TestUtils.java

```diff
@@ -127,6 +129,7 @@ public static MockProject mockProject(@Nullable PicoContainer container) {
     return new MockProject(container, getParentDisposableForCleanup());
   }
 
+  @SuppressFBWarnings(value="DM_STRING_CTOR", justification="Warning due to string concatenation")
```

Comment:
What's the warning/bug in this case?  Is this something we want to blanket disable? Surely findbugs has a way to do that without having to annotate every instance.